src/fn/bga.ts

```diff
@@ -166,6 +168,9 @@ export const bga = (
     (grid.y * p) / 2,
     0.2,
   )
+  if (parameters.noref) {
+    silkscreenRefText.text = ""
+  }
```

Comment:
empty strings are ALWAYS an anti-pattern! not allowed!

`noref` should be implemented in a post-processing transform as discussed in discord